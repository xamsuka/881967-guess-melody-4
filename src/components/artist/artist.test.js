import React from 'react';
import renderer from 'react-test-renderer';
import Artist from './artist.jsx';

const artist = {
  author: `Пелагея`,
  imageSrc: `https://api.adorable.io/avatars/128`,
};

test(`<Artist /> render`, () => {
  const three = renderer
    .create(<Artist author = {artist.author} imageSrc = {artist.imageSrc} indexAnswer = {`answer-1`} />).toJSON();

  expect(three).toMatchSnapshot();
});
