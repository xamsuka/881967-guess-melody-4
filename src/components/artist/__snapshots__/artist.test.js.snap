// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Artist /> render 1`] = `
<div
  className="artist"
>
  <input
    className="artist__input visually-hidden"
    defaultValue="answer-1"
    id="answer-1"
    name="answer"
    type="radio"
  />
  <label
    className="artist__name"
    htmlFor="answer-1"
  >
    <img
      alt="Пелагея"
      className="artist__picture"
      src="https://api.adorable.io/avatars/128"
    />
    Пелагея
  </label>
</div>
`;
